doc/converse/manual.rst

```diff
@@ -2090,6 +2104,56 @@ times affecting performance still apply, This function returns the
 previous (and potentially current, if it was shorter than
 ``newResolution``,) resolution in seconds.
 
+.. _converse_task_queues:
+
+Converse Task Queue
+=====================
+
+In addition to the scheduler queue, Converse provides support for task queues. To enable these queues,
+the flag ``--enable_task_queue`` should be included when Converse is built in SMP mode.
+Converse supports queues managed by the PE's scheduler and those managed by user programs.
```

Comment:
This paragraph should also include a description of what the task queue actually is, what it can be used for, and how the system processes its contents.